### ISSUE TITLE
[All cogs] Tracker / Project Management 

### ISSUE THREAD

**Issue by Vexed01** [Vexed01/Vex-Cogs#8]
# What?
This tracks everything that's planned for my cogs.
If you have a suggestion, please open a new issue or reach out in Discord (Vexed#3211).
If you want to submit a PR for one of these, go ahead! Unless it's small, it's probably best to reach out to me in Discord (Vexed#3211) before starting.

# In Progress 
Status: Multi-incident support

# Planned

### Alias
- [ ] Internal refactoring - split up command into a few more methods 

### AnotherPingCog
- [ ] Let bot owners set global custom emojis (#7) 
- [ ] Let bot owners force the embed version

### GitHub
- [ ] Add milestones support (#6) 

### Status
- [ ] Update event docs 
- [ ] Soft edit-only update when field value changes 
- [ ] 20 May: remove config migration 

### System
...

**Comment by Vexed01** [comment]
Trying to leave a comment?
If you have a suggestion, please open a new issue or reach out in Discord. 
If you want to submit a PR for one of these, go ahead! Unless it's small, it's probably best to reach out to me in Discord before starting.
If it's something else, create a new issue or reach out in Discord.
Discord: Vexed#3211
